slu/tests/test_config/test_config_creation.py

```diff
@@ -52,20 +51,33 @@ def test_api_config_json():
     import os
     import json
 
-    from slu.utils.config import Config, JSONAPIConfigDataProvider
+    from slu.utils.config import Config, OnStartupClientConfigDataProvider
+    from slu import constants as const
     
     with open("config/config.json") as handler:
-        actual_config = json.load(handler)
+        axis_config = json.load(handler)
```

Comment:
I am scared of 57-63.

Reply:
why tho? assumption of files being present/missing?